Add analytics for prospectus link

Euraldius opened the pull request:
https://trello.com/c/f08aGTYi/118-as-a-site-admin-i-want-to-know-if-something-affects-use-of-the-site

Commit:
Add analytics for prospectus link

https://trello.com/c/f08aGTYi/118-as-a-site-admin-i-want-to-know-if-something-affects-use-of-the-site

ghost commented:
I haven't been able to test this code ~ it looks like requests aren't being sent to Google from my local file. Does anyone know how I can test it, or if it's not working because I'm doing something wrong?

jarmstrng commented:
@Euraldius Hm, maybe pushing it to staging for testing is the only way. Let me know if you want to pair on it this weekend

Commit:
Add linker and ecommerce to Google Analytics

We need the linker and ecommerce packages to track information with Tito.

This commit also adds our staging domain to the GA domain list for testing purposes. It will be removed before this
commit is merged into master.

ghost commented:
This PR is no longer relevant.
